trnsysGUI/diagram/Editor.py

```diff
@@ -73,6 +73,7 @@ def __init__(self, parent, projectFolder, jsonPath, loadValue, logger):
         self.projectFolder = projectFolder
 
         self.diagramName = os.path.split(self.projectFolder)[-1] + ".json"
+        self.diagramPath = os.path.join(self.projectFolder, self.diagramName)
```

Comment:
Let's discuss this refactoring choice, as it may lead to incremental improvements.